### PR TITLE
Replace global.alert use to fix eslint warnings

### PR DESCRIPTION
**Pull request by vcalvello**
Replaces `alert` with `Alert.alert` to fix eslint warnings.

Test Plan:
----------
```
yarn lint
```
Check `no-alert` lint warning displayed.

Changelog:
----------
[INTERNAL] [ENHANCEMENT] - Replace `alert` with `Alert.alert` to fix eslint warning.

### CODE REVIEW

File: RNTester/js/AccessibilityIOSExample.js [hunk 1]
```diff
@@ -12,18 +12,20 @@
...
-const {AccessibilityInfo, Text, View, TouchableOpacity} = ReactNative;
+const {AccessibilityInfo, Text, View, TouchableOpacity, Alert} = ReactNative;
 
 class AccessibilityIOSExample extends React.Component<{}> {
   render() {
     return (
       <View>
         <View
-          onAccessibilityTap={() => alert('onAccessibilityTap success')}
+          onAccessibilityTap={() => Alert.alert('onAccessibilityTap success')}
```
Comment (TheSavior) [inline RNTester/js/AccessibilityIOSExample.js]: This is almost the same. global's alert sets the title of the alert to "Alert". https://github.com/facebook/react-native/blob/master/Libraries/Core/setUpAlert.js#L20

**Comment by RSNara** [comment]
What's the rationale behind removing reliance on `global.alert`? It seems like a fairly innocent/standard function that provides the convenience of calling `Alert.alert` in a fairly standardized manner. Considering how all browsers implement this function, I'm not really sure why we should diverge and try to get rid of it. Is that what the intent of this PR is?

**Comment by TheSavior** [comment]
I think the intent is to fix the eslint warning which is pretty standard to disallow alert and console calls. I think the question is do we disable the linter for this file, or throughout all of RNTester, or use the Alert module which doesn’t trigger the lint. *shrug*

**Comment by vcalvello** [comment]
The idea is to fix the eslint warning. I'm not pretty sure how, though. I usually use alert and console calls for debugging purposes and I prefer not to disable linting rules, that's why I chose using `Alert.alert`. Anyway we can disable linting in that line with `/*eslint-disable-line no-alert*/`. Definitely, I wouldn't disable the linter for the whole RNTester project. What do you prefer?

**Pull request closed without merging**